src/db/nodeDb.js

```diff
@@ -259,5 +294,17 @@ module.exports = {
       return result.rows[0].has;
     }
     return false;
-  }
+  },
+
+  insertContractEvent: async (event) => {
+    const { contractTxId, caller, transactionId, sortKey, input, blockHeight, blockTimestamp, data } = event;
+
+    await drePool.query(
+      `
+                INSERT INTO contract_event (contract_tx_id, sort_key, tx_id, caller, input, block_timestamp, block_height, data)
+                VALUES ($1, $2, $3, $4, $5, $6, $7, $8)
+                ON CONFLICT(contract_tx_id, sort_key) DO UPDATE SET data = EXCLUDED.data`,
+      [ contractTxId, sortKey, transactionId, caller, input, blockTimestamp, blockHeight, data ]
```

Comment:
hmm, maybe JSON.stringify before passing input and data to pg :) 
I don't know...